firefox-ios/Client/Frontend/Home/LegacyHomepageViewController.swift

```diff
@@ -183,6 +184,9 @@ class LegacyHomepageViewController: UIViewController,
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.8) { [weak self] in
             self?.displayWallpaperSelector()
         }
+        Task { @MainActor in
+            termsOfUseDelegate?.showTermsOfUse(context: .homepageOpened)
+        }
```

Comment:
`viewDidAppear` executes on the main thread already, do we need the `Task { @MainActor in … }`, or could we call `showTermsOfUse` directly?